### ISSUE TITLE
[Thermal Foundation] Not able to put torches on TF metal blocks

### ISSUE THREAD

**Issue by xypista** [CoFH/Feedback#448]
I can't put any torches, levers, tracks on Thermal Foundation blocks (Tin Block, Copper Block, ...).

Tested with only 4 mods:
CoFHCore-[1.7.10]3.0.2-262.jar
ThermalFoundation-[1.7.10]1.0.0-81.jar
CodeChickenCore-1.7.10-1.0.5.37-universal.jar
NotEnoughItems-1.7.10-1.0.4.95-universal.jar

Edit: It works on ores

**Comment by KingLemming** [comment]
Odd. It works for me. I'm marking as fixed in dev, since it must be.

**Comment by BoboSims** [comment]
Just to be helpful, I tested this with what I'm *certain* are the latest released versions of TD, TE, TF, and Core, thinking that if I could get it to work, something else might be the cause in @xypista's game.

However, I've found that indeed I can **not** put torches or levers on *any* of the *Thermal metal* blocks. I haven't even attempted tracks anymore. So if it works for you, @KingLemming, then it must indeed be because something in your dev version fixes it. :)
